brave/app/_locales/en/messages.json

```diff
@@ -5,15 +5,24 @@
   "braveRewards": {
     "message": "Brave Rewards"
   },
+  "connect": {
```

Comment:
This was present in translations upstream, but just did not appear to have a russian translation entry for it. I've included it here so it gets included in our next transifex round